spec/lib/tasks/import_spec.rb

```diff
@@ -52,4 +52,38 @@
       )
     end
   end
+
+  describe "import:whitehall_documents" do
+    before do
+      allow($stdout).to receive(:puts)
+      Rake::Task["import:whitehall_documents"].reenable
+    end
+
+    it "creates pending whitehall migration document imports" do
+      allow(WhitehallDocumentImportJob).to receive(:perform_later)
+
+      expect {
+        Rake::Task["import:whitehall_documents"].invoke("123 456")
+      }.to change {
+        WhitehallMigration::DocumentImport.pending.exists?(whitehall_document_id: 123)
+      }.to(true).and change {
+        WhitehallMigration::DocumentImport.pending.exists?(whitehall_document_id: 456)
+      }.to(true)
+    end
+
+    it "queues a job for each document to be imported" do
+      Rake::Task["import:whitehall_documents"].invoke("123 456 789")
+      expect(WhitehallDocumentImportJob).to have_been_enqueued.exactly(3).times
+    end
+
+    it "creates a single WhitehallMigration given a single document" do
```

Comment:
I think this test and the one [below](https://github.com/alphagov/content-publisher/pull/1958/files#diff-9c116fd10c62f7c423ebfcf6c4a21937R84) can be merged into one test checking that a single WhitehallMigration has been created as there is no logic here we need to test regarding multiple / single doc imports.

Reply:
Yeah, I did want them in one test, but it kept failing!  Found the issue though - needed to `reenable` the task between calls.  Testing rake tasks eh!